packages/flutter_markdown/lib/src/widget.dart

```diff
@@ -32,6 +32,11 @@ typedef MarkdownOnSelectionChangedCallback = void Function(
 typedef MarkdownTapLinkCallback = void Function(
     String text, String? href, String title);
 
+/// Signature for custom image builder.
```

Comment:
This still needs a lot more context/information; it should explain what it should actually do and how it fits into the overall API surface. See the comments comments above for examples.